tests/e2e/isbservice.go

```diff
@@ -321,17 +321,16 @@ func DeleteISBServiceRollout(name string) {
 	}).WithTimeout(TestTimeout).Should(BeTrue(), "The ISBService should have been deleted but it was found.")
 }
 
-// TODO: replace pipelineRolloutName with an array of names to check that each is paused
-// pipelineRolloutName is pipelinerollout and child pipeline that may be checked to see if it is pausing or not after update
+// pipelineRolloutNames is an array of pipelinerollout names that are checked to see if it is pausing or not after update
 // newSpec is the updated spec of the ISBService defined in the rollout
 // verifySpecFunc is passed to the verifyISBServiceSpec func which verifies the ISBService spec defined in the updated rollout
 // matches what we expect
 // dataLossFieldChanged determines if the change of spec incurs a potential data loss or not, as defined in the USDE config: https://github.com/numaproj/numaplane/blob/main/config/manager/usde-config.yaml
 // recreateFieldChanged determines if the change of spec should result in a recreation of InterstepBufferService and its underlying pipelines - also defined in the USDE config
 // pipelineIsFailed informs us if any dependent pipelines are currently failed and to not check if they are running
 func UpdateISBServiceRollout(
-	isbServiceRolloutName,
-	pipelineRolloutName string,
+	isbServiceRolloutName string,
+	pipelineRolloutNames []string,
```

Comment:
Actually, I think you'd need a new array of struct instead to include:
`pipelineRolloutName`, `pipelineIsFailed`, `overrideSourceVertexReplicas`